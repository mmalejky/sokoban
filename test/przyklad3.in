+
--
-a-
----
-----
------
-c#####
------
-----
----
-b-
--
*

a2
b8
a2
b8
a2
a2
a6
b4
b8
a6
c8
a6
c8
a6
c4
c8
c8
c8
c8
0
.
