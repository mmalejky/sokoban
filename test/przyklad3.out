+
--
-a-
----
-----
------
-c#####
------
-----
----
-b-
--
*
+
--
-@-
-a--
-----
------
-c#####
------
-----
----
-b-
--
+
+
--
---
-a--
-----
------
-c#####
------
-----
-b--
-@-
--
+
+
--
---
-@--
-a---
------
-c#####
------
-----
-b--
---
--
+
+
--
---
----
-a---
------
-c#####
------
-b---
-@--
---
--
+
+
--
---
----
-@---
-a----
-c#####
------
-b---
----
---
--
+
+
--
---
----
-@---
-a----
-c#####
------
-b---
----
---
--
+
+
--
---
----
-----
-@a---
-c#####
------
-b---
----
---
--
+
+
--
---
----
-----
--a---
-c#####
------
b@---
----
---
--
+
+
--
---
----
-----
--a---
-c#####
b-----
@----
----
---
--
+
+
--
---
----
-----
--a---
-c#####
b-----
@----
----
---
--
+
+
--
---
----
-----
-ca---
-@#####
b-----
-----
----
---
--
+
+
--
---
----
-----
-ca---
-@#####
b-----
-----
----
---
--
+
+
--
---
----
-c---
-@a---
--#####
b-----
-----
----
---
--
+
+
--
---
----
-c---
--@a--
--#####
b-----
-----
----
---
--
+
+
--
---
----
c@---
---a--
--#####
b-----
-----
----
---
--
+
+
--
---
c---
@----
---a--
--#####
b-----
-----
----
---
--
+
+
--
c--
@---
-----
---a--
--#####
b-----
-----
----
---
--
+
+
c-
@--
----
-----
---a--
--#####
b-----
-----
----
---
--
+
C
@-
---
----
-----
---a--
--#####
b-----
-----
----
---
--
+
+
c-
@--
----
-----
---a--
--#####
b-----
-----
----
---
--
+
